docs/site/api.md

```diff
@@ -268,6 +268,65 @@ _Corresponding [REPL Snippet](custom-commands.md) variable: `$top-level-defined-
     ```javascript
     const [range, text] = ranges.currentTopLevelForm();
     ```
+## `editor`
+
+The `editor` module has facilites (well, a facility, so far) for editing Clojure documents.
+
+### `editor.replace()`
+
+With `editor.replace()` you can replace a range in a Clojure editor with new text. The arguments are:
+
+* `editor`, a `vscode.TextEditor`
+* `range`, a `vscode.Range`
+* `newText`, a string
+
+=== "Joyride"
+
+    ```clojure
+    (-> (p/let [top-level-form-range (first (calva/ranges.currentTopLevelForm))
+                _ (calva/editor.replace vscode/window.activeTextEditor top-level-form-range "Some new text")]
+          (println "Text replaced!"))
+        (p/catch (fn [e]
+                   (println "Error replacing text:" e))))
+    ```
+
+=== "JavaScript"
+
+    ```javascript
+    const topLevelRange = calvaApi.ranges.currentTopLevelForm();
+    calva.editor.replace(topLevelRange, "Some new text")
+      .then((_) => console.log("Text replaced!"))
+      .catch((e) => console.log("Error replacing text:", e));
+    ```
+
+## `pprint`
+
+The `pprint` module lets you pretty print Clojure code/data using Calva's [pretty printing](pprint.md) engine (which in turn uses [zprint](https://github.com/kkinnear/zprint)).
+
+### `pprint.prettyPrint()`
+
+Use `pprint.prettyPrint()` to pretty print some Clojure data using your Calva pretty printing options. It accepts these arguments:
+
+* `text`, a `string` with the text to pretty print
+* `options`, a JavaScript object with [pretty printing](pprint.md) options, this is optional and will default to the current settings.
+
+The function is synchronous and returns the prettified text.
+
+=== "Joyride"
+
+    ``` clojure
+    (println (calva/pprint.prettyPrint "Some text")))
+    ```
+
+=== "JavaScript"
+
+    ``` javascript
+    console.log(calvaApi.pprint.prettyPrint();
+    ```
+
+### `pprint.prettyPrintingOptions()`
+
+Use to get the current pretty printint options:
```

Comment:
Is there supposed to be an example here after the colon?

Reply:
Also there's a typo in "printing"

Reply:
I got a bit tired of coming up with examples, and figured I could skip it here. 😄 We should think of a way to document the API with a bit less effort than it currently demands.